### ISSUE TITLE
Disable a block of variables

### ISSUE THREAD

**Issue by edjroz** [golang/lint#492]
Is there any way to disable a block of variables as such:
```go
var (
   ErrorA = errors.New("this error happens on...")
   ErrorB = errors.New("this errors happens on ...")
...
)
```

technically you can disable them them by appending `//nolint` at the end of each line but ideally you might want to have the ability to disable the whole block

**Comment by llorephie** [comment]
Same request from me, having ability to switch off linter on block of code will be very useful, as there are such cases as hardware points, symbol tables or simply database columns, and adding comment to each line is some way time burning routine.

**Comment by mvdan** [comment]
Thank you for submitting this issue! As per https://github.com/golang/go/issues/38968, we are freezing and deprecating golint. There's no drop-in replacement to golint per se, but you should find that [Staticcheck](https://staticcheck.io/) works well in encouraging good Go code, much like golint did in the past, since it also includes style checks. There's always `gofmt` and `go vet` too, of course.

If you would like to contribute further, I'd encourage you to engage [Staticcheck's issue tracker](https://github.com/dominikh/go-tools/issues) or look at [vet's open issues](https://github.com/golang/go/issues?q=is%3Aissue+is%3Aopen+cmd%2Fvet+in%3Atitle), as they are both actively maintained. If you have an idea that doesn't fit into either of those tools, you could look at [other Go linters](https://github.com/golangci/awesome-go-linters), or write your own - these days it's fairly straightforward with [go/analysis](https://pkg.go.dev/golang.org/x/tools/go/analysis).

To help avoid confusion, I'm closing all issues before we freeze the repository. If you have any feedback, you can leave a comment on the proposal thread where it was decided to deprecate golint - though note that the proposal has been accepted for nearly a year. Thanks!
